.github/workflows/gke.yml

```diff
@@ -39,6 +39,12 @@ jobs:
         run: |
           sed -i "s/ENVIRONMENT/$ENVIRONMENT/g" deployment.yml
 
+      - name: Set ReCaptcha
+        env:
+          RECAPTCHA_SECRET: "{{ secrets.RECAPTCHA_SITEKEY }}"
+        run: |
+          sed -i "s/RECAPTCHA_SECRET/$RECAPTCHA_SECRET/g" config.example.yml
```

Comment:
It should have a previous step where config.example.yml is copied to config.yml and then the config.yml is overwritten. I
need to tweak the config files though, we should have a config.prod.yml file instead of config.example.yml for that purpose.

Reply:
The config.example.yml is copied to config.yml after, in a build step in Dockerfile.